src/vsg/app/CommandGraph.cpp

```diff
@@ -18,6 +18,10 @@ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLI
 
 using namespace vsg;
 
+CommandGraph::CommandGraph()
```

Comment:
Could this be simply left as =default; in the header?

Reply:
It could, but it doesn't really gain anything.  I like have even empty code blocks there so it's easy to add/remove debug output from the constructors.